RELEASE.txt

```diff
@@ -98,14 +117,12 @@ the corresponding release branch.
   - Open that URL and check for the wheels as
     ``scikit_image-<versiontag>-<platforminfo>.whl``
 
-- Update the docs:
+- Make sure you have all the required write permissions (if not, you will need
+  to ask Stéfan or Juan to grant you the access), specifically, to:
```

Comment:
Possibly this should actually be moved to the very top? It is frustrating to get to this point and then be stuck, especially if on a different timezone.